tests/migrations/test_state.py

```diff
@@ -144,6 +145,7 @@ class Meta:
 
         # The default manager is used in migrations
         self.assertEqual([name for name, mgr in food_state.managers], ['food_mgr'])
+        self.assertEqual([type(name) for name, mgr in food_state.managers], [six.text_type])
```

Comment:
I think I would use `assertTrue(all(isinstance(name, six.text_type) for ...)` but don't care too much either way.